logspout/modules.go

```diff
@@ -6,4 +6,5 @@ import (
 	_ "github.com/gliderlabs/logspout/transports/tcp"
 	_ "github.com/gliderlabs/logspout/transports/udp"
 	_ "github.com/looplab/logspout-logstash"
+        _ "github.com/gliderlabs/logspout/adapters/multiline"
```

Comment:
File should be formated with gofmt. You could also do it online on https://play.golang.org/

Reply:
Done